CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 0.16.2
+
+### fix
+
+-fix(provider-generator): remove duplicates between wrapper classes and interfaces generated from provider schemas [\#2895](https://github.com/hashicorp/terraform-cdk/pull/2895)
```

Comment:
Nitpick but the missing space means the list is not properly formatted.

Reply:
Yeah, good point. Had to add that one by hand 🙈